__tests__/workers/cdc.ts

```diff
@@ -654,6 +679,28 @@ describe('alerts', () => {
   });
 });
 
+describe('feed', () => {
+  type ObjectType = Feed;
+  const base: ChangeObject<ObjectType> = {
+    userId: '1',
+    id: '1',
+  };
+  it('should notify on alerts created', async () => {
+    await expectSuccessfulBackground(
+      worker,
+      mockChangeMessage<ObjectType>({
+        after: base,
+        before: null,
+        op: 'c',
+        table: 'feed',
+      }),
+    );
+    const params = [con, base.userId, { myFeed: 'created' }];
+    expect(updateAlerts).toBeCalledTimes(1);
+    expect(mocked(updateAlerts).mock.calls[0]).toEqual(params);
```

Comment:
Let's test that the DB indeed changed.
I usually mock the pubsub functions because there is no other way of testing it